### PR TITLE
1st draft of the Heap Verification routines.

### PR DESCRIPTION
**Pull request by Don-Ward**
All verification is controlled by an environment variable VRFY.
Some structures, notably lists, are not verified yet.
See rmemmgt.r for details.

### CODE REVIEW

File: src/h/rexterns.h [hunk 1]
```diff
@@ -419,3 +419,9 @@ extern struct  b_cons *LstTmpFiles;
 #ifdef Audio
 extern int isPlaying;
 #endif					/* Audio */
+
+#ifdef VerifyHeap
+extern long vrfy;
+extern void vrfyLog(const char *fmt, ...);
+extern void vrfy_Live_Table(struct b_table *b);
```
Comment (Jafaral) [inline src/h/rexterns.h:426]: Where are these functions? I don't see their implementation as part of this PR?

Comment (Jafaral) [reply src/h/rexterns.h:426]: Found it! Github wasn't showing rmemmgt.r by default because  the diff is large!

**Comment by Jafaral** [comment]
@Don-Ward if you want the code under your new VerifyHeap to be CI tested on github with out messing with the CI test itself , you can do the trick of  defining VerifyHeap as part of the the commit and push it to a temporary branch on your sandbox.

**Comment by Jafaral** [comment]
Just did that here: https://github.com/Jafaral/unicon/commit/59775d1f4d538059a4eb25543ff567dfec056209

**Comment by Jafaral** [comment]
Test failed on Ubuntu: https://github.com/Jafaral/unicon/runs/650353668?check_suite_focus=true

**Comment by Don-Ward** [comment]
The error happens on the Ubuntu build because the verifier code references something that is not defined when we are building iconc (i.e. when COMPILER is defined). I removed the offending code in commit 7eb3098.

A more interesting question is why the macOS build does not build iconc.

**Comment by Jafaral** [comment]
For bookkeeping it is probably good to say why we are closing this one and not merging, something like:

Merged with #49

**Pull request closed without merging**